waspc/src/Wasp/Analyzer/Parser/Parser.y

```diff
@@ -133,27 +133,28 @@ Name :: { ExtImportName }
   | '{' id '}' { ExtImportField $2 }
 
 Quoter :: { Expr }
-  : pos '{=' Quoted pos '=}' {% if $2 /= $5
-                                then throwError $ QuoterDifferentTags ($2, $1) ($5, $4)
-                                else return $ Quoter $2 $3
-                             }
+  : posStart '{=' posEnd Quoted posStart '=}' posEnd
+      {% if $2 /= $6
+         then throwError $ QuoterDifferentTags (WithCtx (ctxFromRgn $1 $3) $2) (WithCtx (ctxFromRgn $5 $7) $6)
+         else return $ Quoter $2 $4
+      }
 Quoted :: { String }
   : quoted        { $1 }
   | Quoted quoted { $1 ++ $2 }
 
--- Special production that returns the position of the token that will get scanned after it.
+-- | Special production that returns the start of the next/following token.
 -- NOTE(martin): You might wonder why does it use position of the last scanned (therefore *previous*)
```

Comment:
hah ok, I was just wondering about that! (kidding :P). But its a great comment for somebody who has the context :D

Reply:
However will be looking at this in the future would probably be confused by this because it is counter-intuitive, so I thought it is best to capture it here and avoid future confusion. It turned out to sound a bit like a story :D.